src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -913,15 +918,32 @@ function IOURequestStepConfirmation({
                     />
                     {(isLoading || isLoadingReceipt) && <FullScreenLoadingIndicator />}
                     {PDFThumbnailView}
-                    <ReceiptDropUI
-                        onDrop={(e) => {
-                            const file = e?.dataTransfer?.files[0];
-                            if (file) {
-                                file.uri = URL.createObjectURL(file);
-                                setReceiptOnDrop(file);
-                            }
-                        }}
-                    />
+                    {/* TODO: remove canUseMultiFilesDragAndDrop check after the feature is enabled */}
+                    {canUseMultiFilesDragAndDrop ? (
+                        <DropZoneUI
+                            onDrop={(e) => {
+                                const file = e?.dataTransfer?.files[0];
+                                if (file) {
+                                    file.uri = URL.createObjectURL(file);
+                                    setReceiptOnDrop(file);
+                                }
```

Comment:
I've seen that the logic is duplicated a bit there, we do this twice here

and we do the same later on (excepting the new state function used) with:
```TS
                                    const file = e?.dataTransfer?.files[0];
                                    if (file) {
                                        file.uri = URL.createObjectURL(file);
                                        setReceiptAndNavigate(file);
                                    }
```

Question: can we create a reusable function to avoid repeating the same code pattern for these 4 occurrences?

Reply:
@lakchote yeah, I can but one of the component will be removed later on so we won't have the code duplication

Reply:
Fine, then disregard this.